src/pure.ts

```diff
@@ -20,5 +20,5 @@ export { waitFor };
 export { waitForElementToBeRemoved };
 export { within, getQueriesForElement };
 export { getDefaultNormalizer };
-export { renderHook };
+export { renderHook, RenderHookOptions, RenderHookResult };
```

Comment:
Export `RenderHookOptions`, `RenderHookResult` as `export type` next to other `export type` declarations.